sdk/terraform/Azure.ResourceManager.Terraform/api/Azure.ResourceManager.Terraform.netstandard2.0.cs

```diff
@@ -0,0 +1,146 @@
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Terraform.Models.ExportResult>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Terraform.Models.ExportResult>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ResourceProvisioningState : System.IEquatable<Azure.ResourceManager.Terraform.Models.ResourceProvisioningState>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ResourceProvisioningState(string value) { throw null; }
+        public static Azure.ResourceManager.Terraform.Models.ResourceProvisioningState Canceled { get { throw null; } }
+        public static Azure.ResourceManager.Terraform.Models.ResourceProvisioningState Failed { get { throw null; } }
+        public static Azure.ResourceManager.Terraform.Models.ResourceProvisioningState Succeeded { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Terraform.Models.ResourceProvisioningState other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Terraform.Models.ResourceProvisioningState left, Azure.ResourceManager.Terraform.Models.ResourceProvisioningState right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Terraform.Models.ResourceProvisioningState (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Terraform.Models.ResourceProvisioningState left, Azure.ResourceManager.Terraform.Models.ResourceProvisioningState right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct TargetProvider : System.IEquatable<Azure.ResourceManager.Terraform.Models.TargetProvider>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public TargetProvider(string value) { throw null; }
+        public static Azure.ResourceManager.Terraform.Models.TargetProvider Azapi { get { throw null; } }
+        public static Azure.ResourceManager.Terraform.Models.TargetProvider Azurerm { get { throw null; } }
```

Comment:
please rename these to have proper casing `AzApi` and `AzureRM` or simply `Arm`.

Reply:
Implemented in https://github.com/Azure/azure-rest-api-specs/pull/31198